Makefile

```diff
@@ -98,6 +98,10 @@ thrift: idl-submodule thrift-image
 	$(THRIFT) -o /data --gen go:$(THRIFT_GO_ARGS) --out /data/$(THRIFT_GEN_DIR) /data/idl/thrift/sampling.thrift
 	$(THRIFT) -o /data --gen go:$(THRIFT_GO_ARGS) --out /data/$(THRIFT_GEN_DIR) /data/idl/thrift/zipkincore.thrift
 	@echo Generate TChannel-Thrift bindings
+	# tchan doesn't support oneway, so to get collector signature, we need to modify the jaeger.thrift file
```

Comment:
how about we just remove the agent entirely for now? nobody is using it and it's an even cleaner change.